examples/keep-alive-framework/src/main/scala/com/mesosphere/usi/helloworld/KeepAlivePodSpecHelper.scala

```diff
@@ -13,7 +13,7 @@ object KeepAlivePodSpecHelper {
   val runSpec: RunTemplate = RunTemplate(
     resourceRequirements = List(ScalarRequirement(ResourceType.CPUS, 0.001), ScalarRequirement(ResourceType.MEM, 32)),
     shellCommand = """echo "Hello, world" && sleep 30""",
-    role = "test"
+    role = "slave_public"
```

Comment:
wait, why?

Reply:
Agreed, we should be able use whichever role we like.